crates/state/src/font_style.rs

```diff
@@ -105,28 +106,26 @@ impl ParseAttribute for FontStyleState {
     fn parse_attribute(
         &mut self,
         attr: freya_native_core::prelude::OwnedAttributeView<CustomAttributeValues>,
-    ) -> Result<(), crate::ParseError> {
+    ) -> Result<(), ParseError> {
         match attr.attribute {
             AttributeName::Color => {
                 if let Some(value) = attr.value.as_text() {
                     // Make an exception for the "inherit" as in this case we don't want to pass
                     //  a color at all but use the inherited one.
                     if value != "inherit" {
-                        self.color = Color::parse(value)?;
+                        self.color = Color::parse_value(value)?
                     }
                 }
             }
             AttributeName::TextShadow => {
                 if let Some(value) = attr.value.as_text() {
-                    self.text_shadows = value
-                        .split_excluding_group(',', '(', ')')
-                        .map(|chunk| TextShadow::parse(chunk).unwrap_or_default())
-                        .collect();
+                    self.text_shadows = TextShadow::parse_values(value, &Token::Comma)?;
```

Comment:
Wouldn't using ( or ) here not make a difference?

Reply:
I don't quite understand how `(` or `)` would be used when parsing `text-shadow` (I haven't found an example of using `text-shadow` with them anywhere either)